### ISSUE TITLE
Updated wrapper for Share Progress pages

### ISSUE THREAD

**Issue by HPaulJohnson** [MozillaFoundation/Advocacy#861]
We need to update the existing "wrapper" template we use to build Share Progress pages. Currently, the available wrappers have old version of logo and out of date footer. 

We should keep this simple, just updating the logo in the header and making the footer consistent with other active websites (opendata, copyright, etc)

CC: @ScottDowne @beccaklam

**Comment by lovegushwa** [comment]
@beccaklam we need to modify the share progress wrapper. p1 is to replace the footer with a CTA that sits below the video (this is totally a hack.) 

@HPaulJohnson when do you need this by? 

<img width="998" alt="screen shot 2017-04-24 at 8 13 34 pm" src="https://cloud.githubusercontent.com/assets/12228435/25367676/105c83a4-292c-11e7-8774-9e5728edddb6.png">

**Comment by ScottDowne** [comment]
I started a wrapper and it's saved in share progress.

**Comment by ScottDowne** [comment]
I have added the logo, in white text, black highlight, and white page background, and added the cta under the main content.

The cta is mostly un styled. I added line height, font size, black on white, and centered it.

I kept everything simple and black and white for now, I'll let Becca decide where to take it..

One thing I noticed is we can and probably want to move the cta with the link above the video.

Right now we have http://share.mozilla.org/352/180554 (no clue if that link is actually viewable by anyone)
